### ISSUE TITLE
Broken Anvil Not Occuring

### ISSUE THREAD

**Issue by Divineaspect** [BrassGoggledCoders/Workshop#3]
Using an Anvil through it's durability doesn't result in a Broken Anvil

**Comment by FreneticScribbler** [comment]
Yes, I know. The event I needed to do this was removed.

**Comment by Divineaspect** [comment]
wait, the event was removed?

**Comment by FreneticScribbler** [comment]
Yes, in the transition between 1.12 and 1.15. I also can't do the Floralchemy cauldron squashing.

Doing either sensibly would probably require a PR to Forge.
